### ISSUE TITLE
What happened

### ISSUE THREAD

**Issue by Boom1053** [zfbx/zdiscord#14]
![image](https://user-images.githubusercontent.com/42920111/135650387-73bf2115-9eb6-41a4-81e8-dd8bb98eec79.png)

Bots been working fine no updates to core or anything. She just died on me!

**Comment by zfbx** [comment]
Yeah not sure why this has been happening, I believe it's something to do with eris or the websocket resource not handling discord packets properly. soon as node16+ update drops for fivem I'll have the resource converted to use discord.js which will run much better and shouldn't have the errors that this one keeps throwing

**Comment by Boom1053** [comment]
Thank you Tony! It fixed itself after a reboot!
